### DISCUSSION TITLE
注册时一直显示Failed to fetch

### DISCUSSION THREAD

**Discussion by Ebjize1997** [walinejs/waline#994] [Q&A] [answered]
<img width="427" alt="image" src="https://user-images.githubusercontent.com/93699577/166626751-7db214cc-70d0-45e1-bfc6-69527359b62e.png">
请问注册时一直显示failed to fetch 要怎么处理呢——

**Comment by lizheming** [comment, marked as answer]
提供下地址

**Reply by Ebjize1997** [reply]
https://waline-get.vercel.app/

这个，非常感谢！实在是麻烦了——

**Reply by lizheming** [reply]
我试了注册和 GitHub 登录等相关功能都是可以的呀

**Reply by Ebjize1997** [reply]
> 提供下地址

？刚刚试了一下注册成功了（呆滞）前面试了很多次都没有成功，手机端也试了（）但是这次突然就成功了，不好意思打扰了！！！
